src/src/services/models/explorer/createAppModel.ts

```diff
@@ -1478,14 +1478,15 @@ function createAppModel(appConfig: IAppInitialConfig) {
         [AlignmentOptionsEnum.RELATIVE_TIME]: alignByRelativeTime,
         [AlignmentOptionsEnum.ABSOLUTE_TIME]: alignByAbsoluteTime,
         [AlignmentOptionsEnum.CUSTOM_METRIC]: alignByCustomMetric,
-        default: () => {
-          throw new Error('Unknown value for X axis alignment');
-        },
```

Comment:
This is a preliminary fix as there was a bug preventing any app from loading correctly (`alignmentConfigs` issue). There might be a better way of handling it, however this does the trick for the time being